content/api/commands/origin.md

```diff
@@ -233,11 +234,41 @@ and the protocol defaults to `https`. When `cy.visit()` is called with the path
 `/history/founder`, the three are concatenated to make
 `https://www.acme.com/history/founder`.
 
+#### Alternative navigation
+
+```js
+// Do things in primary origin...
+
+cy.visit('https://www.acme.com/history/founder')
+
+// The cy.origin block is required to interact with the cross origin page.
```

Comment:
```suggestion
// The cy.origin block is required to interact with the cross-origin page.
```

Reply:
I'm getting the feeling that you want me to add a dash between 'cross' and 'origin' :)

Reply:
Indeed 😅 Thought we should at least be consistent about it and prior usages have it hyphenated.